sda/local.go

```diff
@@ -398,6 +398,11 @@ func (l *LocalTest) NewLocalConode(port int) *Conode {
 
 }
 
+// NewContext creates a context from a ServerIdentity.
+func (l *LocalTest) NewContext(si *network.ServerIdentity, servID ServiceID, manager *serviceManager) *Context {
```

Comment:
There might be a need to add a `serviceManager` later. For the moment it's only used in a test, to avoid adding a method to `Conode` that returns a context. But I can remove `NewContext` and add a `Context` method to `Conode` if you prefer.